src/components/dev-hub/hero-banner.js

```diff
@@ -126,7 +127,7 @@ const HeroBanner = ({
                             }
                             imageWidthOnMobile={imageWidthOnMobile}
                         >
-                            <img src={background} alt="" />
+                            <img src={background} alt={imageAltText} />
```

Comment:
this only adds alt text to the image on mobile. i don't suppose there's a way to add alt text to a background image? https://github.com/mongodb/devhub/blob/c4f795565ddedd16250a447ce3ca618e99e68850/src/components/dev-hub/hero-banner.js#L40

Reply:
Unfortunately there is not via CSS iirc